### ISSUE TITLE
"mods/ModsDB.ini could not be read!" Error on Startup

### ISSUE THREAD

**Issue by MelodicMax** [thesupersonic16/HedgeModManager#47]
When I press Save & Play or just Play, it opens the command prompt like it normally does and says:
[ForcesModLoader] [ERROR] ModsDB.ini could not be read!
I'm not really sure what to do about this. I've tried deleting the ModsDB.ini file and making a new one after loading the loader again, but it always comes to the same error.

**Comment by thesupersonic16** [comment]
Can you upload the ModsFB.ini file here and mention the location you find the file?

**Comment by MelodicMax** [comment]
> Can you upload the ModsFB.ini file here and mention the location you find the file?
When you say ModsFB.ini, you're referring to ModsDB.ini, right?

[ModsDB.ZIP](https://github.com/thesupersonic16/HedgeModManager/files/3727196/ModsDB.ZIP)

The file location is:
C:\Program Files (x86)\Steam\steamapps\common\Sonic Forces\build\main\projects\exec\mods

**Comment by thesupersonic16** [comment]
Yeah. What version of HMM and FML are you using? the location and file are valid.

**Comment by MelodicMax** [comment]
My HMM version is 6.1-021 and my FML version is 6.7.

**Comment by thesupersonic16** [comment]
That's strange, What I'm getting from this is that its failing to even open the ModsDB.ini file in read mode. Was this working before? Also can you try run Steam as administrator and launch the game from there?

**Comment by MelodicMax** [comment]
> That's strange, What I'm getting from this is that its failing to even open the ModsDB.ini file in read mode. Was this working before? Also can you try run Steam as administrator and launch the game from there?

Okay so I got it working again and here's what I did. I'm not sure exactly what it was specifically that fixed it. First, I did a reinstallation of the loader. Then, I closed and re-opened Steam as an administrator. When I opened the loader through Steam it said there was no mods folder and asked to make a new one. I already had one so I assumed it was making a second folder but it ended up just putting a codes.xml file in the pre-existing mods folder. At this point, I had 2 file explorer tabs open. One with the my mods in it and the second was just the codes.xml file. Both shared the same file path though. Finally, I cut and pasted all the mods into the second mod folder, the loader detected them, I pressed save and play and everything worked like normal.
